packages/psss/src/containers/Tables/CountryTable.js

```diff
@@ -90,7 +90,7 @@ const countryColumns = [
   {
     title: 'Date ',
     key: 'weekNumber',
-    width: '190px',
+    width: '190px', // must be same width as SiteSummaryTable name column to align
```

Comment:
I think a comment is more informative than the variable to ensure future developers keep the widths the same.